build_docs/source/conf.py

```diff
@@ -0,0 +1,43 @@
+# Configuration file for the Sphinx documentation builder.
+
+"""
+
+Steps to generate documentation:
+1) cd to /build_docs
+2) run sphinx-apidoc -f -o /source ../masterplan_tools/   - generates project structure
+3) run make html (or .\make html if you don't have make available globally)    - builds html with sphinx structure
+4) newly built documentation will be in /build_docs/build/html
+
+"""
+
+
+import os
+import pathlib
+import sys
+
+sys.path.insert(0, os.path.abspath(str(pathlib.Path(__file__).resolve().parent.parent)))
+sys.path.append(os.path.abspath(str(pathlib.Path(__file__).resolve().parent.parent / "masterplan_tools")))
```

Comment:
`os.path.abspath` is redundant as `pathlib.Path(__file__).resolve()` already absolute returns absolute path